src/main/java/net/snowflake/ingest/streaming/internal/ParquetRowBuffer.java

```diff
@@ -113,7 +113,9 @@ public void setupSchema(List<ColumnMetadata> columns) {
                 column.getCollation(),
                 column.getOrdinal(),
                 null /* fieldId */,
-                parquetType.isPrimitive() ? parquetType.asPrimitiveType() : null));
+                parquetType.isPrimitive() ? parquetType.asPrimitiveType() : null,
+                clientBufferParameters.isEnableDistinctValuesCount(),
```

Comment:
lets set this to false explicitly until BDEC ingestion decides to start using NDV / NV ?

Reply:
All this parameter comes from `InternalParameterProvider` and it's set to false for FDN mode. or do you mean passing `false` as arg right here?

Reply:
i mean passing false hardcoded since we know FDN doesn't want to do NDV / NV (in fact the service will barf if they're sent over). ok to take in next PR

Reply:
Modified.